lib/legacy/polymer.dom.html

```diff
@@ -274,14 +293,15 @@
     /**
      * Returns the local (re-targeted) target for this event.
      *
-     * @return {Node} The local (re-targeted) target for this event.
+     * @return {!EventTarget} The local (re-targeted) target for this event.
      */
     get localTarget() {
       return this.event.target;
     }
 
     /**
      * Returns the `composedPath` for this event.
+     * @return {!Array<EventTarget>} The nodes this event propagated through
```

Comment:
@aomarks it turns out closure does know what `EventTarget` is but i got confused because of the whole getters-are-not-yet-recognised issue. so these don't produce any types, but they pass the closure checks.

i did change some more types here from `Node` to `EventTarget` because that matches the dom spec.